src/main.cpp

```diff
@@ -4952,7 +4952,7 @@ bool ProcessNewBlock(CValidationState& state, CNode* pfrom, CBlock* pblock, CDis
         }
     }
 
-    if (pwalletMain) {
+    if (pwalletMain && masternodeSync.IsSynced()) {
```

Comment:
Thoughts on having this based on `masternodeSync.IsSynced()` or `IsInitialBlockDownload()`. It is entirely possible to send a transaction before the `mnsync` is finished, and I agree that limiting this to one or the other is a net benefit...just want to discuss which would make more sense.

Reply:
Actually I was going for IDB at first. But I think it's better to wait to be fully synced (and get your last incoming TXs) to fire up more TX. Else you are going to make more TX than necessary, incurring fees and blocks bloating.

Reply:
The first thing that happens within `MultiSend()` and `AutoCombineDust()` is checking whether `IsInitialBlockDownload()` is true, and if so they will return false.

Reply:
@presstab You are right, I should move it there. But I still think `masternodeSync.IsSynced()` is superior to `IsInitialBlockDownload()` in this case.